### ISSUE TITLE
Establish usage guidelines for the logo

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/brand#1]
Establish usage guidelines for the logo. As part of this issue, as select the terms under which the logo is licensed. Document these in a file named usage-guidelines.adoc at the root of this project. The license should go in the file named LICENSE so that GitHub recognizes it.

Use these resources for inspiration and guidance:

* https://brand.18f.gov/logo/ (18F)
* https://opensource.org/logo-usage-guidelines (Open Source)
* https://fedoraproject.org/wiki/Logo/UsageGuidelines (Fedora)
* https://about.twitter.com/en_us/company/brand-resources.html (Twitter)
* https://design.firefox.com/photon/visuals/product-identity-assets.html (Firefox)

**Comment by mojavelinux** [comment]
My instinct is to use a Creative Commons license such as CC BY ND (no derivatives). I'd also be okay with CC BY SA (share alike aka copyleft), though I need some guidance on this as to whether it can result in the brand being diluted.

**Comment by wolandscat** [comment]
I have no experience with licenses for logos, but my sense is also CC-BY-ND. It's what we use for our published specifications to protect them (one hopes) from adulteration.

**Comment by marianhlavac** [comment]
I'd be also cool with CC-BY-ND.

**Comment by AFulgens** [comment]
The problem with CC-BY-ND is that what @mojavelinux suggested: it may dilute the brand, as anybody may use it for anything, even not Asciidoctor related. A logo is mostly trademark protected, if it really is part of a brand (the Mercedes insignia in multiple variant is trademarked, as well as the purple of Milka chocolate, or the signal of T-Systems).

The question which should be asked: what is the intended purpose of this logo? Should it be used as file type icon and such, or as a brand logo?

If the first, then CC-BY-ND may be appropriate as well as a pull request towards https://github.com/primer/octicons and similar (although there it would be MIT licensed... so who knows?), not necessarily Github related projects.
If it is intended to be a brand logo, with protection and everything, then it has to be trademarked and a license/usage text crafted by a lawyer from the field (cf. the Twitter guidelines, they are pretty elaborate. Same holds for Facebook, IBM, etc. They take the logo usage pretty seriously).

IANAL, but I would suggest that until this issue is effectively resolved, a preliminary license should be crafted (anybody has a trademark lawyer friend?), which states that it is OK to use the logo for asciidoctor-related projects (mostly on github anyways), as long as they are linked to this issue (or another, to-be-created-collection-issue in this repository). This way a small branding may already be started, and still controlled, so later on any issues may be rectified.

**Comment by minad** [comment]
@mojavelinux Is it ok to use the rescaled icon for the adoc filetype? https://github.com/Alexander-Miller/treemacs/pull/278

**Comment by minad** [comment]
@mojavelinux Ping. Btw, emacs also uses asciidoctor for checking the files via flycheck. So it makes sense to show this logo in the filetree for adoc files.

**Comment by mojavelinux** [comment]
@minad Yes, this is definitely acceptable. Although we haven't written up the official wording yet, let this be notification of intent of that wording. It's fine to use the Asciidoctor logo as is to represent the project or the AsciiDoc file format. Where we'll probably lock things down a bit is in how it can be remixed / modified, so as not to dilute the brand. But yes, please feel free to use it as just described.

**Comment by mojavelinux** [comment]
I agree it needs to be made clear that the logo should be used to represent Asciidoctor and the AsciiDoc file format alone and nothing else. But aside from a clause like that, I like CC-BY-ND within that context (is that even possible?). We definitely don't want to restrict the usage of the logo as is. It's our traveling gnome. It should go out into the world and see things. But when it does, it's important that it represents us.

I don't think we have to be quite as strict as Fedora about how the logo is used. For example, if the text needs to be made a bit less or more bold, or the color has to be slightly tinted, I'm not super concerned about that. That only puts the burden on us to police this thing.

In the end, I think this just comes down to borrowing some language from one of the sites I referenced above, removing the parts we feel are too restrictive / fussy, and putting it up on the asciidoctor.org site.

**Comment by minad** [comment]
Cool :+1: I don't think being overly strict is necessary, in particular judging from how logos are used between different open source projects. To minimize the effort and to avoid confusion I would probably just go with some established license like CC-BY-ND.

**Comment by HarshKapadia2** [comment]
Can a license please be added to this repo @mojavelinux? That will make using the logo easier, as permissions to use the logo can be linked with the license on this repo.

I wanted to add a footnote like [this one](https://github.com/HarshKapadia2/git_basics/blob/main/README.md):
![image](https://user-images.githubusercontent.com/50140864/147378899-3ffdbc10-2dad-4936-8d82-90de562d1376.png)
